src/Servers/Kestrel/samples/QuicSampleApp/Program.cs

```diff
@@ -34,9 +33,9 @@ public static void Main(string[] args)
                      factory.AddConsole();
                  })
                  .UseKestrel()
-                 .UseMsQuic(options =>
+                 .UseQuic(options =>
                  {
-                     options.Certificate = cert;
+                     options.Certificate = null;
```

Comment:
This can still be set to `cert`, right?

Reply:
Because I'm including the stub msquic, I'm going to swap to use null certs for everything.